devhyun05/EstateChain
# Transactions page

### Features
1. Added List of Work Experiences Table
2. Transferred Adding of Experience in a Button w/ modal


### How has this been tested:
1. supply  necessary .env variables (See .env.example)

- REACT_APP_COMPANY_WALLET_ADDRESS=<company wallet address>
- REACT_APP_CONTRACT_ADDRESS=<contract address> <- You can see this if you run the **deploy script**, inside the terminal 
- REACT_APP_ETH_PROVIDER=<eth-provider> <- You can also see this when you run the **hardhat**

3. from backend (npx hardhat compile, npx hardhat node, npx hardhat run scripts/deploy.js --network localhost)
4. from frontend (yarn start)
5. Must login an metamask account with eth money (0.0002 charge per transaction)
6. Click Add Work Experience Button
7. Add Work Experience 
8. Notice that the experience is now being added in the table

### Screenshots
<img width="1287" alt="image" src="https://github.com/devhyun05/Accuwork/assets/33895507/c0678025-0732-48cb-825b-5dffa920bc63">
